### PR TITLE
[Az.EventHub] Get-AzEventHub accepts pipeline input from Get-AzEventHubNamespace.

### PR DESCRIPTION
**Pull request by hakashya**
## Description

Get-AzEventHub would fail to accept pipeline input from Get-AzEventHubNamespace as described here: #13606. The cmdlet is modified now to successfully take the pipeline input from Get-AzEventHubNamespace as per the guidelines specified [here](https://github.com/Azure/azure-powershell/blob/master/documentation/development-docs/design-guidelines/piping-best-practices.md#using-the--inputobject-parameter).

## Checklist

- [x] I have read the [_Submitting Changes_](../blob/master/CONTRIBUTING.md#submitting-changes) section of [`CONTRIBUTING.md`](../blob/master/CONTRIBUTING.md)
- [x] The title of the PR is clear and informative
- [x] The appropriate `ChangeLog.md` file(s) has been updated:
    - For any service, the `ChangeLog.md` file can be found at `src/{{SERVICE}}/{{SERVICE}}/ChangeLog.md`
    - A snippet outlining the change(s) made in the PR should be written under the `## Upcoming Release` header -- no new version header should be added
- [x] The PR does not introduce [breaking changes](../blob/master/documentation/breaking-changes/breaking-changes-definition.md)
- [x] If applicable, the changes made in the PR have proper test coverage
- [x] For public API changes to cmdlets:
    - [N/A] a cmdlet design review was approved for the changes in [this repository](https://github.com/Azure/azure-powershell-cmdlet-review-pr) (_Microsoft internal only_)
        - {Please put the link here}
    - [x] the markdown help files have been regenerated using the commands listed [here](../blob/master/documentation/development-docs/help-generation.md#updating-all-markdown-files-in-a-module)

### CODE REVIEW

File: src/EventHub/EventHub/Cmdlets/EventHub/GetAzureEventHubs.cs [hunk 1]
```diff
@@ -25,34 +27,45 @@ namespace Microsoft.Azure.Commands.EventHub.Commands.EventHub
     /// <para> If EventHub name provided, a single EventHub detials will be returned</para>
     /// <para> If EventHub name not provided, list of EventHub will be returned</para>
     /// </summary>
-    [Cmdlet("Get", ResourceManager.Common.AzureRMConstants.AzureRMPrefix + "EventHub"), OutputType(typeof(PSEventHubAttributes))]
+    [Cmdlet("Get", ResourceManager.Common.AzureRMConstants.AzureRMPrefix + "EventHub", DefaultParameterSetName = EventhubPropertiesParameterSet), OutputType(typeof(PSEventHubAttributes))]
     public class GetAzureRmEventHub : AzureEventHubsCmdletBase
     {
-        [Parameter(Mandatory = true, ValueFromPipelineByPropertyName = true, Position = 0, HelpMessage = "Resource Group Name")]
+        [Parameter(Mandatory = true, Position = 0, HelpMessage = "Resource Group Name", ParameterSetName= EventhubPropertiesParameterSet)]
```
Comment (wyunchi-ms) [inline src/EventHub/EventHub/Cmdlets/EventHub/GetAzureEventHubs.cs:33]: Lost the definition for parameter set `NamespaceInputObjectParameterSet`.

Comment (hakashya) [reply src/EventHub/EventHub/Cmdlets/EventHub/GetAzureEventHubs.cs:33]: I introduced NamespaceInputObjectParameterSet and EventHubPropertiesParameterSet. Doing so eliminated the _AllParameterSets that was being evaluated originally. The functionality exhibited by the cmdlet earlier was to evaluate _AllParameterSet. This same functionality is now exhibited by EventHubPropertiesParameterSet, thereby not having any breaking changes (although static analyzer considers this to be breaking change due to the loss of _AllParameterSets). 

The introduction of the two parameter sets was to ensure that InputObject was accepted via the pipeline for the cmdlet.

**Comment by msJinLei** [comment]
@hakashya could you suppress the false positive breaking changes in your PR?

**Comment by hakashya** [comment]
@msJinLei Thank you! I've suppressed the false positive breaking changes in the new commit.

File: tools/StaticAnalysis/Exceptions/Az.EventHub/BreakingChangeIssues.csv [hunk 2]
```diff
@@ -0,0 +1,2 @@
+AssemblyFileName,ClassName,Target,Severity,ProblemId,Description,Remediation
```
Comment (msJinLei) [inline tools/StaticAnalysis/Exceptions/Az.EventHub/BreakingChangeIssues.csv:1]: Please keep the double quotes, refer to https://github.com/Azure/azure-powershell/blob/master/tools/StaticAnalysis/Exceptions/Az.Automation/BreakingChangeIssues.csv

**Review by msJinLei — CHANGES_REQUESTED** [review]

File: tools/StaticAnalysis/Exceptions/Az.EventHub/BreakingChangeIssues.csv (hunk 2 above)
Comment (hakashya) [reply tools/StaticAnalysis/Exceptions/Az.EventHub/BreakingChangeIssues.csv]: Thank you, the file is modified to retain double quotes.

**Review by msJinLei — APPROVED** [review]

**Pull request merged**